Deceive/Utils.cs

```diff
@@ -1,23 +1,25 @@
 ﻿using Microsoft.Win32;
 using Microsoft.WindowsAPICodePack.Dialogs;
-using System.Management;
 using System;
+using System.Collections.Generic;
 using System.Diagnostics;
 using System.IO;
 using System.Linq;
+using System.Management;
+using System.Net;
+using System.Reflection;
+using System.Text;
 using System.Text.RegularExpressions;
 using System.Windows.Forms;
-using System.Collections.Generic;
-using System.Reflection;
 
 namespace Deceive
 {
     class Utils
     {
         public static readonly string DATA_DIR = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData), "Deceive");
-        private static Regex AUTH_TOKEN_REGEX = new Regex("\"--remoting-auth-token=(.+?)\"");
-        private static Regex PORT_REGEX = new Regex("\"--app-port=(\\d+?)\"");
-        private static string CONFIG_PATH = Path.Combine(DATA_DIR, "lcuPath");
+        private static readonly Regex AUTH_TOKEN_REGEX = new Regex("\"--remoting-auth-token=(.+?)\"");
+        private static readonly Regex PORT_REGEX = new Regex("\"--app-port=(\\d+?)\"");
```

Comment:
Heh. Looks like these regexes were from a copy-paste of my common utils class, now they finally get used.

Reply:
^^